rolldown.config.js

```diff
@@ -23,6 +23,7 @@ export default defineConfig({
     }
   ],
   platform: 'browser',
+  // @ts-expect-error TODO
   external: (id) => !id.startsWith('.') && !isAbsolute(id),
```

Comment:
![image](https://github.com/user-attachments/assets/42c83589-d5ef-4e6a-b22d-b1b96c61af28)


```suggestion
  external: (/** @type {string} */ id) => !id.startsWith('.') && !isAbsolute(id),
```